### DISCUSSION TITLE
v1.6.2

### DISCUSSION THREAD

**Discussion by lazy-immortal** [lazy-immortal/Sesame#20] [Announcements]
## 更新说明
* 解除 自行打包的版本限制(现在用户可以自行fork项目打包而不受版本限制)
* 优化 庄园帮喂小鸡(帮喂次数达到上限时，不会再发送请求)
* 优化 庄园帮喂小鸡(帮喂的小鸡与自己同在一个家庭时，会调用家庭帮喂)
* 优化 新村丢肥料(丢肥料达到上限时，不会再发送请求)
* 添加 神奇物种-自动收集历史物种(存在未收集的历史物种时会自动兑换历史物种卡道具)
* 调整 神奇物种配置项(合并自动收集历史物种、自动合成图鉴勋章为图鉴勋章)
* 添加 自定义路线列表(如果选择多条，则会随机走其中一条路线)
* 调整 扩展功能(自定义路线)为添加自定义路线到自定义列表
* 增加 按默认顺序派遣动物伙伴选项 by @JiangAshao in https://github.com/lazy-immortal/Sesame/pull/12
* 修复 派遣伙伴默认列表第一个不可派遣 by @JiangAshao in https://github.com/lazy-immortal/Sesame/pull/15
* 合并 派遣动物选项
* 添加 新UI弹窗描述 by @wh-990624

## 下载说明
- Normal 为正常版本，适用于**Android 8.0**及以上的系统
- Compatible 为兼容版本，适用于**Android 7.0**及以下的系统，最低支持**Android 5.1**

## New Contributors
* @JiangAshao made their first contribution in https://github.com/lazy-immortal/Sesame/pull/12

<hr /><em>This discussion was created from the release <a href='https://github.com/lazy-immortal/Sesame/releases/tag/v1.6.2'>v1.6.2</a>.</em>

**Comment by Abysslyz** [comment]
在芝麻粒中设置完成后打开支付宝显示芝麻关门：四十大盗，就是不运行。我是哪里没搞对吗？
